src/Core/Managed/Shared/DataContracts/AggregatedMetricTelemetry.cs

```diff
@@ -0,0 +1,157 @@
+﻿namespace Microsoft.ApplicationInsights.DataContracts
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.External;
+
+    /// <summary>
+    /// Telemetry type used to track aggregated metric information.
+    /// </summary>
+    public sealed class AggregatedMetricTelemetry : ITelemetry, ISupportProperties
+    {
+        internal const string TelemetryName = "Metric";
+
+        internal readonly string BaseType = typeof(MetricData).Name;
+
+        internal readonly MetricData Data;
+        internal readonly DataPoint Metric;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AggregatedMetricTelemetry"/> class with empty 
+        /// properties.
+        /// </summary>
+        public AggregatedMetricTelemetry()
+        {
+            this.Data = new MetricData();
+            this.Metric = new DataPoint();
+            this.Context = new TelemetryContext(this.Data.properties);
+
+            // We always have a single 'metric'.
+            this.Data.metrics.Add(this.Metric);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AggregatedMetricTelemetry"/> class with properties provided.
+        /// </summary>
+        /// <remarks>
+        /// Metric statistics provided are assumed to be calculated over a period of time equaling 1 minute.
+        /// </remarks>
+        /// <param name="metricName">Metric name.</param>
+        /// <param name="count">Count of values taken during aggregation interval.</param>
+        /// <param name="sum">Sum of values taken during aggregation interval.</param>
+        /// <param name="min">Minimum value taken during aggregation interval.</param>
+        /// <param name="max">Maximum of values taken during aggregation interval.</param>
+        /// <param name="standardDeviation">Standard deviation of values taken during aggregation interval.</param>
+        public AggregatedMetricTelemetry(
```

Comment:
I expose parameter-less ctor that can be used along with property setting to populate whatever properties I like. I long considered adding others or making, say, min/max optional in some way on the ctor that takes all properties, but ruled that out at the end. I want user to explicitly state what they want to do and not omit anything. I think the schema I have right now is best, but we can discuss offline.